src/libs/blueprint/conduit_blueprint_mesh_partition.hpp

```diff
@@ -573,6 +584,65 @@ class CONDUIT_BLUEPRINT_API Partitioner
                             std::vector<int> &dest_domain,
                             std::vector<int> &offsets);
 
+    /**
+     @brief Generates a map from adjset vertex ids to their containing chunks.
+            This map is generated per-adjset and is stored in CSR format, and
+            is called by build_interdomain_adjsets() to construct splits of the
+            original adjsets.
+
+     @param chunk_offsets The vector containing starting chunk indices for each
+                          rank.
+     @param chunks A map of pre-partition domains to the set of chunk ids it is
+                   decomposed into, as well as the associated vertex maps for
+                   each chunk.
+     @param domain_map A map of domain ids to the corresponding Conduit node
+                       containing that domain's data.
+     @param[out] adjset_chunk_maps An array of nodes containing mappings from
+                                   shared vertices in adjsets to output chunks.
+     @note Reimplemented in parallel
+     */
+    virtual void get_prelb_adjset_maps(const std::vector<int>& chunk_offsets,
```

Comment:
Does "prelb" mean "pre-loadbalance"?

Reply:
I think it refers to this: https://www.galco.com/buy/Plasti-Bond-a-RobRoy-Company/PRELB-AL-1X90

Reply:
Conduit needs more Elbows.